core/capabilities/ccip/ccipevm/msghasher_test.go

```diff
@@ -349,6 +354,97 @@ func TestMessagerHasher_againstRmnSharedVector(t *testing.T) {
 		require.Equal(t, rmnMsgHash, msgH.String(), "rmn hash and my hash should match")
 	})
 
+	t.Run("solana vector", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("solana", func(t *testing.T) {
```

Only use "vector" for shared test cases w/ RMN

Reply:
Curious if I should add the RMN hash check ? or it's not supported yet ?

Reply:
Not sure - @winder is RMN support in for Solana?

Reply:
Yes and no. The on-chain code is supposed to verify RMN signatures when they are provided with the merkle root, but our implementation doesn't include signatures for the singleton merkle root reports.

Messages originating on Solana do not require an RMN signature.

Support for both of these may be a requirement in the future.

Reply:
So that means there should be an RMN implementation for hashing Solana messages somewhere, when thats complete we should have shared test vectors w/ them.